usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -12735,6 +12735,7 @@ static ep11_target_info_t *get_target_info(STDLL_TokData_t *tokdata)
     target_info = *((void * volatile *)&ep11_data->target_info);
     if (target_info == NULL) {
         TRACE_ERROR("%s: target_info is NULL\n", __func__);
+        pthread_rwlock_unlock(&ep11_data->target_rwlock);
         return NULL;
```

Comment:
Below there is an error check for unlock:

> if (pthread_rwlock_unlock(&ep11_data->target_rwlock) != 0)

Would it make sense to also check and issue a TRACE_DEVEL in case the unlock fails?

Reply:
Right. I added the trace stmt there.